### PR TITLE
Add cargo-testify record

### PR DESCRIPTION
**Pull request by greyblake**
Add record for [cargo-testify](https://github.com/greyblake/cargo-testify) plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ See also [Friends of Rust](https://www.rust-lang.org/friends.html) (organization
   * [ozkriff/zemeroth](https://github.com/ozkriff/zemeroth) — a small 2D turn-based hexagonal strategy game [<img src="https://api.travis-ci.org/ozkriff/zemeroth.svg?branch=master">](https://travis-ci.org/ozkriff/zemeroth)
   * [rsaarelm/magog](https://github.com/rsaarelm/magog) - A roguelike game in Rust
   * [cristicbz/rust-doom](https://github.com/cristicbz/rust-doom) - A renderer for Doom, may progress to being a playable game [<img src="https://api.travis-ci.org/cristicbz/rust-doom.svg?branch=master">](https://travis-ci.org/cristicbz/rust-doom)
-  * [Thinkofname/rust-quake](https://github.com/Thinkofname/rust-quake) - Quake map renderer in Rust 
+  * [Thinkofname/rust-quake](https://github.com/Thinkofname/rust-quake) - Quake map renderer in Rust
```
Comment (greyblake) [inline README.md]: My editor automatically trims trailing spaces, but it must be OK for this PR.

**Comment by luciusmagn** [comment]
The thing with trailing spaces is not a problem, however, the travis badge link needs to be fixed. travis.org part to api.travis.org to prevent redirect. Then the build will succeed and I can merge

**Comment by greyblake** [comment]
@luciusmagn Sorry, overlooked. Fixed in ff699ba

**Comment by luciusmagn** [comment]
Awesome, A-OK now. Thanks for the addition

**Comment by greyblake** [comment]
Thanks :+1:

**Pull request merged**